cves/kernel/CVE-2020-26088.yml

```diff
@@ -75,7 +78,8 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs:
+- https://bugzilla.suse.com/show_bug.cgi?id=1176990
```

Comment:
Again not the ID

Reply:
Modified this similarly to the other vulnerability, by removing it :(